samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/EventLogBean.java

```diff
@@ -126,5 +128,14 @@ public void setFilteredUser(String filteredUser) {
    public void setAssessments(List<SelectItem> assessments) {
       this.assessments = assessments;
    }
+   public Map<Long, Integer> getStatusMap() {
+	   return statusMap;
+   }
+   public void setStatusMap(Map<Long, Integer> statusMap) {
+	   this.statusMap = statusMap;
+   }
+   public boolean isDeleted(Long id){
+	   return statusMap.get(id)==2;
```

Comment:
What is `2` here? Should probably be a constant with a name clarifying it's role/purpose/etc.